packages/styling/src/styles/theme.ts

```diff
@@ -159,45 +160,40 @@ function _makeSemanticColorsFromPalette(
 
     disabledBackground: p.neutralLighter,
     disabledText: p.neutralTertiary,
-    disabledBodyText: p.neutralTertiaryAlt,
+    disabledBodyText: p.neutralTertiary,
     disabledSubtext: p.neutralQuaternary,
+    disabledBodySubtext: p.neutralTertiaryAlt,
 
-    focusBorder: p.black,
+    focusBorder: p.neutralSecondary,
     variantBorder: p.neutralLight,
     variantBorderHovered: p.neutralTertiary,
-    defaultStateBackground: p.neutralLight,
+    defaultStateBackground: p.neutralLighterAlt,
 
     errorText: !isInverted ? p.redDark : '#ff5f5f',
     warningText: !isInverted ? '#333333' : '#ffffff',
-    errorBackground: !isInverted
-      ? 'rgba(232, 17, 35, .2)'
-      : 'rgba(232, 17, 35, .5)',
-    blockingBackground: !isInverted
-      ? 'rgba(234, 67, 0, .2)'
-      : 'rgba(234, 67, 0, .5)',
-    warningBackground: !isInverted
-      ? 'rgba(255, 185, 0, .2)'
-      : 'rgba(255, 251, 0, .6)',
+    errorBackground: !isInverted ? 'rgba(232, 17, 35, .2)' : 'rgba(232, 17, 35, .5)',
```

Comment:
lol is it just me or does prettier keep changing its mind about whether this stuff should be one line or multiple